applications/graphql/main.tf

```
@@ -20,6 +20,10 @@ variable "logger" {
   default     = null
 }
 
+data "aws_ssm_parameter" "airtable_api_key" {
+  name = "/${var.name}/airtable/api-key"
```

Since there's just one API key (rather than application-specific keys, like we're able to do for things like Contentful or Algolia), I'd placed this in a top-level `/airtable/api-key` parameter:

```suggestion
  name = "/airtable/api-key"
```